app/vlselect/main.go

```diff
@@ -73,10 +76,30 @@ var vmuiFileServer = http.FileServer(http.FS(vmuiFiles))
 func RequestHandler(w http.ResponseWriter, r *http.Request) bool {
 	path := r.URL.Path
 
-	if !strings.HasPrefix(path, "/select/") && !strings.HasPrefix(path, "/internal/select/") {
-		// Skip requests, which do not start with /select/, since these aren't our requests.
+	switch {
+	case strings.HasPrefix(path, "/internal/select/") || path == "/internal/select":
+		if *disableInternal {
```

Comment:
It is better to disallow access to `/internal/select/*` when `-select.disable` command-line flag is set too, since this is what most users expect.